docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -183,58 +190,61 @@ The proposer will set the re-proposed block's `Header.Timestamp` as the `Proposa
-In `BFTTime`, if a validator’s clock was behind, the [validator added 1 millisecond to the previous block’s time and used that in its vote messages](https://github.com/cometbft/cometbft/blob/e8013281281985e3ada7819f42502b09623d24a0/internal/consensus/state.go#L2246).
-A goal of adding proposer-based timestamps is to enforce some degree of clock synchronization, so having a mechanism that completely ignores the Unix time of the validator time no longer works.
+In `BFT Time`, if a validator’s clock was behind, the [validator added 1 millisecond to the previous block’s time and used that in its vote messages](https://github.com/cometbft/cometbft/blob/1f430f51f0e390cd7c789ba9b1e9b35846e34642/internal/consensus/state.go#L2460).
+A goal of adding PBTS is to enforce some degree of clock synchronization, so having a mechanism that completely ignores the Unix time of the validator time no longer works.
 Validator clocks will not be perfectly in sync.
 Therefore, the proposer’s current known Unix time may be less than the previous block's `Header.Time`.
 If the proposer’s current known Unix time is less than the previous block's `Header.Time`, the proposer will sleep until its known Unix time exceeds it.
 
-This change will require amending the [defaultDecideProposal](https://github.com/cometbft/cometbft/blob/822893615564cb20b002dd5cf3b42b8d364cb7d9/internal/consensus/state.go#L1180) method.
+This change will require amending the [`defaultDecideProposal`](https://github.com/cometbft/cometbft/blob/1f430f51f0e390cd7c789ba9b1e9b35846e34642/internal/consensus/state.go#L1195) method.
 This method should now schedule a timeout that fires when the proposer’s time is greater than the previous block's `Header.Time`.
 When the timeout fires, the proposer will finally issue the `Proposal` message.
 
 ### Changes to proposal validation rules
 
-The rules for validating a proposed block will be modified to implement proposer-based timestamps.
+The rules for validating a proposed block will be modified to implement PBTS.
 We will change the validation logic to ensure that a proposal is `timely`.
 
-Per the proposer-based timestamps spec, `timely` only needs to be checked if a block has not received a +2/3 majority of `Prevotes` in a round.
+Per the PBTS spec, `timely` only needs to be checked if a block has not received a +2/3 majority of `Prevotes` in a round.
 If a block previously received a +2/3 majority of prevotes in a previous round, then +2/3 of the voting power considered the block's timestamp near enough to their own currently known Unix time in that round.
 
 The validation logic will be updated to check `timely` for blocks that did not previously receive +2/3 prevotes in a round.
-Receiving +2/3 prevotes in a round is frequently referred to as a 'polka' and we will use this term for simplicity.
+Receiving +2/3 prevotes in a round is frequently referred to as a 'Polka' and we will use this term for simplicity.
```

Comment:
Can we move this sentence to the place in the text where the term Polka is first used?

Reply:
Sure, didn't notice that inconsistency in the original text.